extract.ts

```diff
@@ -16,7 +16,7 @@ function hash (str) {
   return (h >>> 0).toString(36)
 }
 
-function extract (raw, ext, key) {
+export const extract = (raw: string, ext: string, key: string) => {
```

Comment:
Is there any benefit in converting these to anonymous functions? I've kinda always preferred using the `function` keyword for any named functions so that files are easier to scan (so not everything is a `const`, but I don't have a strong opinion.

Reply:
Afaik the advantages are automatic binding of `this` and implicit returns when not using braces to create a code block. It's a style choice I suppose.